ghedesigner/building.py

```diff
@@ -12,5 +12,5 @@ def add_heat_pump(self, heat_pump: HeatPump):
     def calculate(self, simulation_time: int, inlet_temperature: float, flow_rate: float) -> float:
         outlet_temperature = inlet_temperature
         for heat_pump in self.heat_pumps:
-            outlet_temperature = heat_pump.calculate(simulation_time, inlet_temperature, flow_rate)
+            outlet_temperature = heat_pump.calculate(simulation_time, outlet_temperature, flow_rate)
```

Comment:
I don't think this is used anywhere yet (@Myoldmopar  can confirm) but, we will want to pass `inlet_temperature` in, and return `outlet_temperature`. I think you can roll this change back.

Reply:
I get what you are doing here @axelstudios.  It looks like you are passing in the outlet temperature, but really you are just passing in the "latest" outlet temperature.  We initialize the outlet temperature to the first inlet temperature, then loop over each heat pump, passing the "latest" outlet temperature in to subsequent heat pumps.  If we imagine the heat pumps being in series, then I ----think---- that you have made a good correction here.  But if we imagine the heat pumps in parallel, or something else, then this function needs more than just your change.  We would need to add mixing in, etc.  What do you think @mitchute ?

Reply:
Oh, yeah... it looks like I've got a case of the Monday-mornings... All good then, at least for now. Proceed.